pkg/apis/apps/types.go

```diff
@@ -44,6 +44,21 @@ type StatefulSet struct {
 	Status StatefulSetStatus
 }
 
+// PodCreationPolicyType defines the policy for creating pods under a stateful set.
+type PodCreationPolicyType string
+
+const (
+	// OrderedReadyPodCreation will create pods in strictly increasing order on
+	// scale up and strictly decreasing order on scale down, progressing only when
+	// the previous pod is ready or terminated. At most one pod will be changed
+	// at any time.
+	OrderedReadyPodCreation PodCreationPolicyType = "OrderedReady"
+	// ParallelPodCreation will create and delete pods as soon as the stateful set
```

Comment:
It has a wikipedia page. In all seriousness though, as per our API conventions, we don't exclude acronyms: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#naming-conventions